From ff1773a21904d678bc46f8cb4f97f0b1e2c65568 Mon Sep 17 00:00:00 2001
From: Mohammad Rafi Shaik <mohammad.rafi.shaik@oss.qualcomm.com>
Date: Sun, 20 Jul 2025 21:28:44 +0530
Subject: [PATCH] FROMLIST: ucm2: Qualcomm: Rename qcs6490-rb3gen2 and
 qcs9075-iq-evk ucm2 conf

Rename the ucm2 conf for Qualcomm qcs6490-rb3gen2 and qcs9075-iq-evk.

qcs6490-rb3gen2-snd-card.conf -> QCS6490-RB3Gen2.conf
qcs9075-iq-evk-snd-card.conf -> LEMANS-EVK.conf

Removed snd-card tags from conf files and update the
mixer settings for LEMANS-EVK.

Signed-off-by: Mohammad Rafi Shaik <mohammad.rafi.shaik@oss.qualcomm.com>
Upstream-Status: Submitted [https://github.com/alsa-project/alsa-ucm-conf/pull/596]
---
 .../{qcs6490-rb3gen2-snd-card.conf => QCS6490-RB3Gen2.conf}  | 0
 .../qcs9075-iq-evk-snd-card => qcs9100/lemans-evk}/HiFi.conf | 5 +++--
 .../lemans-evk/LEMANS-EVK.conf}                              | 2 +-
 .../{qcs6490-rb3gen2-snd-card.conf => QCS6490-RB3Gen2.conf}  | 0
 .../qcs9075-iq-evk-snd-card.conf => qcs9100/LEMANS-EVK.conf} | 2 +-
 5 files changed, 5 insertions(+), 4 deletions(-)
 rename ucm2/Qualcomm/qcs6490/QCS6490-RB3Gen2/{qcs6490-rb3gen2-snd-card.conf => QCS6490-RB3Gen2.conf} (100%)
 rename ucm2/Qualcomm/{qcs9075/qcs9075-iq-evk-snd-card => qcs9100/lemans-evk}/HiFi.conf (73%)
 rename ucm2/Qualcomm/{qcs9075/qcs9075-iq-evk-snd-card/qcs9075-iq-evk-snd-card.conf => qcs9100/lemans-evk/LEMANS-EVK.conf} (52%)
 rename ucm2/conf.d/qcs6490/{qcs6490-rb3gen2-snd-card.conf => QCS6490-RB3Gen2.conf} (100%)
 rename ucm2/conf.d/{qcs9075/qcs9075-iq-evk-snd-card.conf => qcs9100/LEMANS-EVK.conf} (52%)

diff --git a/ucm2/Qualcomm/qcs6490/QCS6490-RB3Gen2/qcs6490-rb3gen2-snd-card.conf b/ucm2/Qualcomm/qcs6490/QCS6490-RB3Gen2/QCS6490-RB3Gen2.conf
similarity index 100%
rename from ucm2/Qualcomm/qcs6490/QCS6490-RB3Gen2/qcs6490-rb3gen2-snd-card.conf
rename to ucm2/Qualcomm/qcs6490/QCS6490-RB3Gen2/QCS6490-RB3Gen2.conf
diff --git a/ucm2/Qualcomm/qcs9075/qcs9075-iq-evk-snd-card/HiFi.conf b/ucm2/Qualcomm/qcs9100/lemans-evk/HiFi.conf
similarity index 73%
rename from ucm2/Qualcomm/qcs9075/qcs9075-iq-evk-snd-card/HiFi.conf
rename to ucm2/Qualcomm/qcs9100/lemans-evk/HiFi.conf
index 6673324..5915d1c 100644
--- a/ucm2/Qualcomm/qcs9075/qcs9075-iq-evk-snd-card/HiFi.conf
+++ b/ucm2/Qualcomm/qcs9100/lemans-evk/HiFi.conf
@@ -3,8 +3,8 @@ SectionVerb {
 		TQ "HiFi"
 	}
 	EnableSequence [
-		cset "name='PRIMARY_SDR_MI2S_RX Audio Mixer MULTIMEDIA0' 1"
-		cset "name='MULTIMEDIA1 Audio Mixer TERTIARY_SDR_MI2S_TX' 1"
+		cset "name='PRIMARY_MI2S_RX Audio Mixer MultiMedia1' 1"
+		cset "name='MultiMedia2 Mixer TERTIARY_MI2S_TX' 1"
 	]
 }
 
@@ -25,5 +25,6 @@ SectionDevice."Mic" {
 	Value {
 		CapturePriority 100
 		CapturePCM "hw:${CardId},1"
+		CaptureChannels 1
 	}
 }
diff --git a/ucm2/Qualcomm/qcs9075/qcs9075-iq-evk-snd-card/qcs9075-iq-evk-snd-card.conf b/ucm2/Qualcomm/qcs9100/lemans-evk/LEMANS-EVK.conf
similarity index 52%
rename from ucm2/Qualcomm/qcs9075/qcs9075-iq-evk-snd-card/qcs9075-iq-evk-snd-card.conf
rename to ucm2/Qualcomm/qcs9100/lemans-evk/LEMANS-EVK.conf
index 669ba6b..71e8c91 100644
--- a/ucm2/Qualcomm/qcs9075/qcs9075-iq-evk-snd-card/qcs9075-iq-evk-snd-card.conf
+++ b/ucm2/Qualcomm/qcs9100/lemans-evk/LEMANS-EVK.conf
@@ -1,6 +1,6 @@
 Syntax 4
 
 SectionUseCase."HiFi" {
-	File "/Qualcomm/qcs9075/qcs9075-iq-evk-snd-card/HiFi.conf"
+	File "/Qualcomm/qcs9100/lemans-evk/HiFi.conf"
 	Comment "HiFi quality Music"
 }
diff --git a/ucm2/conf.d/qcs6490/qcs6490-rb3gen2-snd-card.conf b/ucm2/conf.d/qcs6490/QCS6490-RB3Gen2.conf
similarity index 100%
rename from ucm2/conf.d/qcs6490/qcs6490-rb3gen2-snd-card.conf
rename to ucm2/conf.d/qcs6490/QCS6490-RB3Gen2.conf
diff --git a/ucm2/conf.d/qcs9075/qcs9075-iq-evk-snd-card.conf b/ucm2/conf.d/qcs9100/LEMANS-EVK.conf
similarity index 52%
rename from ucm2/conf.d/qcs9075/qcs9075-iq-evk-snd-card.conf
rename to ucm2/conf.d/qcs9100/LEMANS-EVK.conf
index 1e49d38..e458c25 100644
--- a/ucm2/conf.d/qcs9075/qcs9075-iq-evk-snd-card.conf
+++ b/ucm2/conf.d/qcs9100/LEMANS-EVK.conf
@@ -1,6 +1,6 @@
 Syntax 4
 
 SectionUseCase."HiFi" {
-	File "/Qualcomm/qcs9075/qcs9075-iq-evk-snd-card/HiFi.conf"
+	File "/Qualcomm/qcs9100/lemans-evk/HiFi.conf"
 	Comment "HiFi quality Music."
 }
-- 
2.34.1

